src/componenets/CardList.tsx

```diff
@@ -0,0 +1,28 @@
+import Card from "./Card";
+import "./CardList.css";
+
+interface CardListInterface {
+  id: number;
+  created_at: string;
+  updated_at: null;
+  url: string;
+  title: string;
+  description: string;
+  image_source: string;
+  folder_id: number;
+}
+
+type CardListProps = {
+  selectedFolderData: CardListInterface[];
```

Comment:
보통 배열 데이터 타입은 이런식으로 단일 타입명을 배열로 표현해주는 편입니다~
```suggestion
  selectedFolderData: Card[];
```